ansible/workers.yml

```
@@ -286,19 +305,7 @@
       state: installed
     when: ansible_distribution != 'RedHat' or ansible_distribution_version.split(".")[0] == "7"
```

```suggestion
    when: ansible_distribution == 'RedHat' and ansible_distribution_version.split(".")[0] == "7"
```
I tried the playbook on Fedora and it's trying to execute this piece because the logic is wrong. Can you fix it?